src/Actions/ConfirmPassword.php

```diff
@@ -32,7 +32,7 @@ public function __invoke(StatefulGuard $guard, $user, ?string $password = null)
      * @param  string|null  $password
      * @return bool
      */
-    protected function confirmPasswordUsingCustomCallback($user, ?string $password = null)
+    protected function confirmPasswordUsingCustomCallback($user, #[\SensitiveParameter] ?string $password = null)
```

Comment:
This version requires PHP 8.1 as minimum and `SensitiveParameter` is available on PHP 8.2 and above.

Need to require https://github.com/symfony/polyfill-php82 if we want to merge this

Reply:
Ah thanks! I'll close. 🙇‍♂️